eras/alonzo/impl/cardano-ledger-alonzo.cabal

```diff
@@ -138,10 +138,12 @@ library testlib
         cardano-ledger-core:{cardano-ledger-core, testlib},
         cardano-ledger-shelley:{cardano-ledger-shelley, testlib},
         cardano-strict-containers,
+        plutus-core,
```

Comment:
We should never depend on `plutus-core`, instead all of the types relevant to ledger can be imported through `plutus-ledger-api`
```suggestion
```